components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -89,6 +90,11 @@ struct SiteInfo {
   // Web page specific fields, if available and allowed
   string? title;
   string? hostname;
+
+  // Note: iOS doesn't support optional int32s un mojom, so we use -1 to
```

Comment:
cc @kylehickinson can we fix this? Using -1 is not ideal here

Reply:
We can't support it in a good way unfortunately, Obj-C doesn't support optional primitives, so the only way to do it is by putting the number into a `NSNumber` box, but then you lose type information at the translation layer (Swift only gets to see `NSNumber`, not `int32`) so no way to have compile time checks that you're providing the correct number type  when passing a `NSNumber` back through to Obj-C/C++

Reply:
then -1 should at least be a constant defined here. What is this content id supposed to be for anyway @fallaciousreasoning ? Can you add comments explaining?

Reply:
I've updated the comment explaining what this is. I'm not sure what you mean by making this a constant defined here - can we define constants in mojom?